application.json

```diff
@@ -1180,7 +1180,7 @@
+                      "exploreParams": "marketing_analytics/fb_adset_date_fact?fields=adset.name,campaign.name,fact.average_click_rate,parent_fact.average_click_rate,fact.average_click_rate_delta_ratio&f[campaign.name]=&f[fact.click_rate_good]=Yes&f[fact.period]=28+day&f[fact.date_period_latest]=Yes&f[fact.date_date]=28+days&f[fact.adset_name]=&sorts=fact.average_click_rate_delta_ratio+desc&limit=50&vis=%7B%22type%22%3A%22table%22%2C%22show_view_names%22%3Afalse%2C%22show_row_numbers%22%3Atrue%2C%22truncate_column_names%22%3Afalse%2C%22hide_totals%22%3Afalse%2C%22hide_row_totals%22%3Afalse%2C%22table_theme%22%3A%22gray%22%2C%22limit_displayed_rows%22%3Afalse%2C%22enable_conditional_formatting%22%3Afalse%2C%22conditional_formatting_include_totals%22%3Afalse%2C%22conditional_formatting_include_nulls%22%3Afalse%2C%22series_labels%22%3A%7B%22fact.average_click_rate%22%3A%22Conversion+Rate%22%2C%22parent_fact.average_click_rate%22%3A%22Campaign+Conversion+Rate%22%2C%22fact.average_click_rate_delta_ratio%22%3A%22Conversion+Rate+Delta%22%7D%7D&filter_config=%7B%22campaign.name%22%3A%5B%7B%22type%22%3A%22%3D%22%2C%22values%22%3A%5B%7B%22constant%22%3A%22%22%7D%2C%7B%7D%5D%2C%22id%22%3A9%2C%22error%22%3Afalse%7D%5D%2C%22fact.click_rate_good%22%3A%5B%7B%22type%22%3A%22is%22%2C%22values%22%3A%5B%7B%22constant%22%3A%22Yes%22%7D%2C%7B%7D%5D%2C%22id%22%3A10%2C%22error%22%3Afalse%7D%5D%2C%22fact.period%22%3A%5B%7B%22type%22%3A%22is%22%2C%22values%22%3A%5B%7B%22constant%22%3A%2228+day%22%7D%2C%7B%7D%5D%2C%22id%22%3A11%2C%22error%22%3Afalse%7D%5D%2C%22fact.date_period_latest%22%3A%5B%7B%22type%22%3A%22is%22%2C%22values%22%3A%5B%7B%22constant%22%3A%22Yes%22%7D%2C%7B%7D%5D%2C%22id%22%3A0%2C%22error%22%3Afalse%7D%5D%2C%22fact.date_date%22%3A%5B%7B%22type%22%3A%22past%22%2C%22values%22%3A%5B%7B%22constant%22%3A%2228%22%2C%22unit%22%3A%22day%22%7D%2C%7B%7D%5D%2C%22id%22%3A1%2C%22error%22%3Afalse%7D%5D%2C%22fact.adset_name%22%3A%5B%7B%22type%22%3A%22%3D%22%2C%22values%22%3A%5B%7B%22constant%22%3A%22%22%7D%2C%7B%7D%5D%2C%22id%22%3A13%2C%22error%22%3Afalse%7D%5D%7D&dynamic_fields=%5B%5D&origin=share-expanded",
```

Comment:
what is the change here?

Reply:
it was incorrectly filtering date to "Quarter"